example.py

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/env python3
 """This example can be run safely as it won't change anything in your box configuration."""
+import sys
 
-import asyncio
-import logging
+if "http" in sys.modules:
+    raise ImportError("Crawler must be imported before http module")
+import http.cookies
 
-from aiosysbus import AIOSysbus
-from aiosysbus.exceptions import (
+http.cookies._is_legal_key = lambda _: True
```

Comment:
This is taken from https://github.com/aio-libs/aiohttp/issues/2683
Maybe there is another way... but I'm not sure!

Reply:
I also doubt that this hack would be accepted in Home Assistant.

Reply:
I've reworked this to parse the "Set-cookie" header manually, and then inject the "Cookie" header manually